lib/src/core/vincenty_distance_calculation.dart

```diff
@@ -2,8 +2,6 @@ import 'core.dart';
 
 /// Vincenty formula for Geodesic Distance Calculation
 class VincentyDistance {
-  /// Calculate the geodesic distance between two points 
```

Comment:
Please add it back.

info - lib/src/core/vincenty_distance_calculation.dart:5:17 - Missing documentation for a public member. Try adding documentation for the member. - public_member_api_docs